vector/src/main/java/im/vector/app/features/home/room/detail/RoomDetailFragment.kt

```diff
@@ -478,6 +482,23 @@ class RoomDetailFragment @Inject constructor(
         }
     }
 
+    private fun setupEmojiPopup() {
+        EmojiPopup
+                .Builder
+                .fromRootView(rootConstraintLayout)
+                .setKeyboardAnimationStyle(R.style.emoji_fade_animation_style)
+                .setOnEmojiPopupShownListener { composerLayout.composerEmojiButton.setImageResource(R.drawable.ic_keyboard) }
+                .setOnEmojiPopupDismissListener { composerLayout.composerEmojiButton.setImageResource(R.drawable.ic_insert_emoji) }
+                .build(composerLayout.composerEditText)
+                .also {
+                    emojiPopup = it
```

Comment:
Why not just affecting the value like this: `val emojiPopup = EmojiPopup.Builder...`?
It seems that `emojiPopup` does not have to be a member of the class.